### PR TITLE
Passing information between CalendarView and CDK Datepicker + Datepicker Input

### PR DESCRIPTION
**Pull request by victoriaaa234**
Added checks for CalendarView in the CDK datepicker tests

### CODE REVIEW

File: src/cdk/datetime/datepicker-input.ts [hunk 1]
```diff
@@ -139,6 +139,11 @@ export class CdkDatepickerInput<D> implements AfterContentInit, ControlValueAcce
   set min(value: D | null) {
     this._min = this._getValidDateOrNull(this._dateAdapter.deserialize(value));
     this._validatorOnChange();
+    if (this._datepicker) {
```
Comment (jelbourn) [inline src/cdk/datetime/datepicker-input.ts]:
```ts
if (this._datepicker && this._datepicker.view) {
  ...
}
```

(here and below)

Comment (victoriaaa234) [reply src/cdk/datetime/datepicker-input.ts]: done

File: src/cdk/datetime/datepicker.ts [hunk 2]
```diff
@@ -72,19 +76,27 @@ export class CdkDatepicker<D> implements OnDestroy {
...
     return this._datepickerInput && this._datepickerInput.max;
   }
+  set maxDate(date: D | null) {
+    if (this.view) {
+      this.view.maxDate = date;
+    }
+  }
 
   /** The filter function used to determine which dates are selectable. */
   get _dateFilter(): (date: D | null) => boolean {
```
Comment (mmalerba) [inline src/cdk/datetime/datepicker.ts]: shouldn't dateFilter have a similar setup?

Comment (victoriaaa234) [reply src/cdk/datetime/datepicker.ts]: done

**Pull request merged**